AGPs/AGP-Aracon2020.md

```diff
@@ -0,0 +1,80 @@
+---
+AGP: N/A
+Title: Aracon 2020
+Author: @stefanobernardi
+Status: Stage III
+Track: Finance
+Created: 2019-10-01
+---
+
+# AGP-AC: AraCon 2020
+
+## Address of the transfer recipient
+
+- An ad-hoc AraCon DAO, or the Events DAO. The Aragon Association will provide an address.
+
+## Amount of the transfer
+
+- Up to €295k equivalent of DAI
+
+## Number and frequency of transfers if recurring
+
+- One time transfer
+Unused funds will be returned to the Aragon Association's multisig (`0xcafe1a77...`)
+- Fiat expenses would be incurred by the Association and reimbursed by the DAO to the AA DAO
```

Comment:
Would prefer this to be more specific about what "the DAO" refers to and what the "AA DAO" refers to (for people who lack context, and for the sake of precision).

Reply:
Fixed